user_data/strategies/Strategy002.py

```diff
@@ -30,10 +28,10 @@ class Strategy002(IStrategy):
     # Minimal ROI designed for the strategy.
     # This attribute will be overridden if the config file contains "minimal_roi"
     minimal_roi = {
-        "60":  0.01,
-        "30":  0.03,
-        "20":  0.04,
-        "0":  0.05
+        "60": 0.01,
```

Comment:
can we please revert all the space changes on this pr
- another thing we would catch if we pre-review our own pr